app/src/main/java/org/simple/clinic/drugs/EditMedicinesEffect.kt

```diff
@@ -18,4 +18,6 @@ object GoBackToPatientSummary : EditMedicinesEffect()
 
```

Comment:
Can we rename this commit to `Go back to the previous screen after medicines are refilled`